### ISSUE TITLE
I tried with my laptop too, exactly the same result. It has Windows 10 Home v.2004.

### ISSUE THREAD

**Issue by laciba96** [laciba96/WinBox-Reloaded#3]
I tried with my laptop too, exactly the same result. It has Windows 10 Home v.2004.

Something different did happen when creating VM, this error popped up while it still went through with it:
![image](https://user-images.githubusercontent.com/80648513/118693252-92b29a80-b813-11eb-83fd-1fb30c4407dc.png)

_Originally posted by @ts-korhonen in https://github.com/laciba96/WinBox-Reloaded/issues/1#issuecomment-843363491_

**Comment by laciba96** [comment]
Can you give me more information about, how did you get this error?
I mean simply the fields you specified in the wizard like:

- name, working directory, 
- the contents of this directory if it was existed before the wizard
- the template you chose, and the options (if there was any)
- and the way you specified the disk (if you used the wizard, the fields of the wizard)

By a quick test, the only way I could reprodruce this error, is that:
1. I started the disk wizard by the "Change" button
2. I cleared the file name, and canceled the dialog
3. Clicked through the new VM wizard, and got the same screen as you
This behaviour I found will be fixed, but I want to know there is any there any other method to produce the same message? What other conditions can cause this, if there any?

**Comment by ts-korhonen** [comment]
I just clicked through the new VM rather quickly right after first time running after installing. The settings are just the ones that are selected if I just click next. I think the error popped up right before it started to download 86Box and the download window was over it.

I tried it again two times and the latter brought in the error again. It looks like it's related to downloading 86Box (I deleted the folder so it would download it again). This time download failed to create the 86Box folder and the same error pops up if I try to press Next to finish.

Also trying again it gives the error on finish dialog when there is no 86Box folder. Checking for emulator updates fixes it.

I think the first time it didn't download the emulator soon enough and that's why it showed the error once.
